### PR TITLE
install grunt

### PR DESCRIPTION
**Pull request by laurafitzgerald**
**JIRA**
https://issues.jboss.org/browse/AGDIGGER-170

**Changes**
Install grunt

### CODE REVIEW

**Comment by laurafitzgerald** [comment]
@aliok could you take a look at this when you get some time?

**Comment by laurafitzgerald** [comment]
@odra pushed update change to include fastlane dependencies. Tested via this [pr](https://github.com/aerogear/digger-jenkins/pull/77)

File: Dockerfile [hunk 1]
```diff
@@ -42,7 +50,11 @@ RUN curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.33.2/install.sh
     echo 'CI=Y' >> ${HOME}/.bashrc && \
     nvm install ${NODEJS_DEFAULT_VERSION} && \
     chmod -R 777 ${NVM_DIR} && \
-    npm install -g cordova@${CORDOVA_DEFAULT_VERSION}
+    npm install -g cordova@${CORDOVA_DEFAULT_VERSION} && \
+    gem install fastlane@${FASTLANE_DEFAULT_VERSION} && \
```
Comment (odra) [inline Dockerfile]: Correct way to specify version in gem command: `gem install fastlane -v ${FASTLANE_DEFAULT_VERSION}`

**Comment by aliok** [comment]
👁

**Comment by aliok** [comment]
I was able to build the image, go into it and execute grunt and fastlane commands. I haven't done further tests as I need a Grunt project and a fastlane project.

@laurafitzgerald  For testing Grunt, I will do these (will take some time):
* build the image
* feed the image to Digger (use it in OpenShift)
* get the Grunt based template: https://github.com/feedhenry-templates/feedhenry-cordova-sync-app
* build it

**Comment by laurafitzgerald** [comment]
@aliok thanks for the review. I was able to build [this Android](https://github.com/laurafitzgerald/helloworld-android-gradle) application (using the fastlane tool) using this new image [here](https://rhmap.osm1.skunkhenry.com/#projects/z5jhxrvebpuseejvt2ux5vdg/apps/z5jhxrvhskbjkqcsmokynmyg/build)
If you are happy I will go ahead and merge, re-tag and push the image?

**Comment by odra** [comment]
Tested it locally and I was able to use both grunt and fastlane 👍

**Comment by aliok** [comment]
Tested on OSM1. I only tested the Grunt part though.
All good, feel free to merge

**Pull request merged**